scanpy_scripts/exchangeable_loom.py

```diff
@@ -336,9 +336,9 @@ def write_exchangeable_loom(adata, filename, col_graphs=['neighbors']):
                     sce_path = '@colGraphs${}'.format(path)
                 else:
                     if loom_path.startswith('/.attrs['):
-                        path = loom_path[8:-1]
-                    elif loom_path.startswith('/global/'):
-                        path = loom_path[8:]
+                        path = loom_path[8:-1] # remove '/.attrs['
+                    elif loom_path.startswith('/attr/'):
+                        path = loom_path[6:] # remove '/attr/'
```

Comment:
Starting from 6 as opposed to 8 is correct here right? No need to explain, just flagging it in case it was a typo.

Reply:
Yes, was removing "/global/" now "/attr/", so 2 characters short.